Python/gc.c

```diff
@@ -1467,7 +1487,12 @@ gc_collect_increment(PyThreadState *tstate, struct gc_collection_stats *stats)
     gc_list_validate_space(&survivors, gcstate->visited_space);
     gc_list_merge(&survivors, visited);
     assert(gc_list_is_empty(&increment));
-    gcstate->work_to_do += gcstate->heap_size / SCAN_RATE_DIVISOR / scale_factor;
+    Py_ssize_t delta = (gcstate->heap_size - gcstate->prior_heap_size)*3;
```

Comment:
Binary operations, where two operands require spaces, may be more standard.
`(gcstate->heap_size - gcstate->prior_heap_size) * 3`